spec/policies/restaurant_policy_spec.rb

```diff
@@ -8,8 +8,6 @@
   let(:user) { create(:user) }
   let(:user_owner) { create(:user, email: 'owner@test.com', owner: true) }
 
```

Comment:
!!!! not related to this line
You uploaded some pictures from uploads before added it to gitignore
1 - remove the uploads folder from gitignore
2 - remove your files from uploads
3 - commit this deleting
4 - add uploads folder to gitignore again